src/number_theory/liouville/liouville_number.lean

```diff
@@ -142,51 +142,52 @@ calc (1 - 1 / m)⁻¹ * (1 / m ^ (n + 1)!) ≤ 2 * (1 / m ^ (n + 1)!) :
   end
 ... = 1 / (m ^ n!) ^ n : congr_arg ((/) 1) (pow_mul m n! n)
 
+/--  An upper estimate on the remainder. This estimate works with `m ∈ ℝ` satisfying `2 ≤ m` and is
+weaker than the estimate `liouville_number.remainder_lt'` above. However, this estimate is
+more useful for the proof. -/
+lemma remainder_lt (n : ℕ) {m : ℝ} (m2 : 2 ≤  m) :
+  remainder m n < 1 / (m ^ n!) ^ n :=
+(remainder_lt' n $ one_lt_two.trans_le m2).trans_le (aux_calc _ m2)
+
 /-!  Starting from here, we specialize to the case in which `m` is a natural number. -/
 
 /--  The sum of the `k` initial terms of the Liouville number to base `m` is a ratio of natural
 numbers where the denominator is `m ^ k!`. -/
-lemma liouville_number_rat_initial_terms {m : ℕ} (hm : 0 < m) (k : ℕ) :
-∃ p : ℕ, liouville_number_initial_terms m k = p / m ^ k! :=
+lemma partial_sum_eq_rat {m : ℕ} (hm : 0 < m) (k : ℕ) :
```

Comment:
How about `initial_sum_eq_rat`, using `initial, instead of `partial`?

Reply:
Wikipedia https://en.wikipedia.org/wiki/Series_(mathematics) calls this a partial sum of a series.

Reply:
Ok, leave it as is: I think that I might be the only one who is confused about "partial/initial"!